src/Resources/config/app/routing/admin/product_block.yml

```diff
@@ -0,0 +1,51 @@
+lakion_sylius_cms_admin_product_block:
+    resource: |
+        alias: lakion_sylius_cms.product_block
+        section: admin
+        templates: SyliusAdminBundle:Crud
+        except: ['show', 'update', 'delete']
```

Comment:
Why do you using except instead of only? `except: ['show', 'update', 'delete']` is longer than `only: ['index', 'create']`